geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/ingest/ConverterIngest.scala

```diff
@@ -42,6 +44,14 @@ class ConverterIngest(dsParams: Map[String, String],
     ds.createSchema(sft)
   }
 
+  val factory = new BasePooledObjectFactory[SimpleFeatureConverter[_]] {
+    override def wrap(obj: SimpleFeatureConverter[_]) = new DefaultPooledObject[SimpleFeatureConverter[_]](obj)
+    override def create(): SimpleFeatureConverter[_] = SimpleFeatureConverters.build(sft, converterConfig)
+  }
+
+  private val converterPool =
+    new GenericObjectPool[SimpleFeatureConverter[_]](factory)
```

Comment:
presumably the pool size is unbounded? would there ever be a case where a thread had to wait for a pooled object?